modules/hello-world/partials/installation.adoc

```diff
@@ -5,6 +5,43 @@ The Couchbase C SDK can be installed via apt or yum repositories on GNU/Linux; h
 It may also be https://github.com/couchbase/libcouchbase[built from source^] on any of the platforms mentioned above, and more.
 
 
+== Installing on Linux
+
+For installation on GNU/Linux, install the _couchbase-release_ repository, and then install the _libcouchbase_ packages.
+The following examples download and install _couchbase-release_ repository, a C and C++ compiler, and the C SDK core (_libcouchbase3_), command line tools (_libcouchbase2-tools_), and the development tools (_libcouchbase-devel_ [RPM] or _libcouchbase-dev_ [DEB]).
+
+////
+
+// the server repo only carries LCB 2.n - 
+// put this section back in when it changes in a future release
+
+.Debian and Ubuntu
+[source,bash]
+----
+# Only needed during first-time setup:
+wget http://packages.couchbase.com/releases/couchbase-release/couchbase-release-1.0-7-amd64.deb
+sudo dpkg -i couchbase-release-1.0-7-amd64.deb
+# Will install or upgrade packages
+sudo apt-get update
+sudo apt-get install libcouchbase-dev libcouchbase2-bin build-essential
+----
+
+Note that as of Couchbase Data Platform 6.5, _couchbase-release_ (1.0.7) supports Debian 7, 8, 9, and 10, and Ubuntu 14.04, 16.04, and 18.04. If you wish to install a Couchbase SDK on another Ubuntu or Debian variation, follow the steps below for manual repository configuration, or for installing directly from binary or source (but note that while newer releases may work, other platforms are not necessarily supported).
+
+.RHEL and CentOS
+[source,bash]
+----
+# Only needed during first-time setup:
+wget http://packages.couchbase.com/releases/couchbase-release/couchbase-release-1.0-7-x86_64.rpm
```

Comment:
we publish libcouchbase3 packages into different repositories, and server release package does not install those files repo/list files

Current repo files for RPMs hosted here:
```
https://packages.couchbase.com/clients/c/repos/rpm/libcouchbase-amzn2-x86_64.repo
https://packages.couchbase.com/clients/c/repos/rpm/libcouchbase-centos7-x86_64.repo
https://packages.couchbase.com/clients/c/repos/rpm/libcouchbase-centos8-x86_64.repo
```

Current list files for DEBs hosted here:
```
https://packages.couchbase.com/clients/c/repos/deb/libcouchbase-debian9.list
https://packages.couchbase.com/clients/c/repos/deb/libcouchbase-debian10.list
https://packages.couchbase.com/clients/c/repos/deb/libcouchbase-ubuntu1604.list
https://packages.couchbase.com/clients/c/repos/deb/libcouchbase-ubuntu1804.list
https://packages.couchbase.com/clients/c/repos/deb/libcouchbase-ubuntu2004.list
```

I would not recommend SDK users to go with `couchbase-release` for libcouchbase 3 at least, as that package is intended for Couchbase Server, not for SDK. And our team does not control which keys and repository descriptors installed using it. We have section that describes how to configure repositories for libcouchbase in this partial already. And this is supported way to install libcouchbase.